tests/ui/suggestions/deref-path-method.rs

```diff
@@ -1,6 +1,6 @@
 fn main() {
     let vec = Vec::new();
     Vec::contains(&vec, &0);
-    //~^ ERROR no function or associated item named `contains` found for struct `Vec<_, _>` in the current scope
+    //~^ ERROR `Vec<_, _>` is not an iterator [E0599]
     //~| HELP the function `contains` is implemented on `[_]`
 }
```

Comment:
Bit of an unfortunate suggestion change for an unstable method. This was added in https://github.com/rust-lang/rust/pull/100302, cc @compiler-errors since this will no longer test what it was intended to.

Reply:
pls file a bug, we probably can filter by stability or sth idk